pepper-apis/docs/specification/src/components/request-bodies.yml

```diff
@@ -182,3 +182,29 @@ UpdateActivityPropertiesRequestBody:
         properties:
           index:
             type: integer
+RegisterGovernedRequestBody:
+  description: user data
+  required: false
+  content:
+    application/json:
+      schema:
+        type: object
+        properties:
+          firstName:
+            type: string
+            description: user's first name, if available
+          lastName:
+            type: string
+            description: user's last name, if available
+          languageCode:
+            type: string
+            description: user's preferred language, as an ISO 639-1 code
+            example: en
+          timeZone:
+            type: string
+            description: user's time zone, as an IANA time zone region identifier
+            example: America/New_York
+          alias:
+            type: string
+            description: governed user's alias (name, etc.)
+            example: John Doe
```

Comment:
As discussed, we're not going to support setting `alias` just yet. Let's remove this from the docs.

Reply:
Updated the docs